docs/mash-referrals/index.md

```diff
@@ -0,0 +1,30 @@
+---
+id: introduction
+title: MASH Referrals
+slug: /mash-referrals
+---
+
+Last updated: 24th January 2022
+
+For children's social care, we had started development on moving the process of managing and working on MASH referrals into the social care case management system. As of writing this, the programme decided not to continue this work.
+
+We are in the process of re-purposing and cleaning up the code and infrastructure setup for the MASH work. We have not finished deciding what gets removed or re-purposed and because of this, we want to keep track of all the resources related to MASH, where they live and what decisions we have made regarding them.
+
+Decisions should be documented in the [Architecture Decision Records](../decisions).
+
+## Table of MASH related resources
```

Comment:
Jira ticket mentioned work in progress branches. Are those documented in relevant repos or are there actually any of them at the moment?

Reply:
Great idea to keep track of all this 👍